pkg/depsolvednf/depsolvednf.go

```diff
@@ -252,6 +254,11 @@ func collectRepos(pkgSets []rpmmd.PackageSet) []rpmmd.RepoConfig {
 	return repos
 }
 
+// Set the SBOM type to generate with the depsolve.
+func (s *Solver) SBOMType(sbomType sbom.StandardType) {
+	s.sbomType = sbomType
+}
```

Comment:
This should have been `SetSBOMType` because ultimately this makes impossible to create a getter: https://go.dev/doc/effective_go#Getters